pkg/shell-operator/debug_server.go

```diff
@@ -14,14 +14,14 @@ import (
 	"github.com/flant/shell-operator/pkg/task/dump"
 )
 
-func DefaultDebugServer() *debug.Server {
+func newDebugServer() *debug.Server {
```

Comment:
debug_server methods are used during bootstrap in addon_operator 

https://github.com/flant/addon-operator/blob/e4887ca4cd1b248ea7ec6d6f326d96cf77ea9a32/pkg/addon-operator/bootstrap.go#L45
https://github.com/flant/addon-operator/blob/e4887ca4cd1b248ea7ec6d6f326d96cf77ea9a32/pkg/addon-operator/bootstrap.go#L73